app/Controllers/Http/DefinitionsController.ts

```diff
@@ -0,0 +1,158 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import { getUnixTimestamp } from 'App/Helpers/Customs'
+import Definition from 'App/Models/Definition'
+import CreateDefinitionValidator from 'App/Validators/CreateDefinitionValidator'
+import { DateTime } from 'luxon'
+
+export default class DefinitionsController {
+  public async index({ params, response }: HttpContextContract) {
+    const { term } = params
+    try {
+      const definitions = await Definition.query()
+        .preload('user')
+        .preload('category')
+        .where('status_definition_id', 2)
+        .where('term', 'like', `${term}`)
+
+      if (definitions.length === 0) {
+        return response.status(404).json({
+          code: 404,
+          status: 'Not Found',
+          message: 'Term not found',
+        })
+      }
+
+      return response.json({
+        code: 200,
+        status: 'Success',
+        message: 'Definitions found',
+        data: definitions.map((data) => {
+          const { id, term, definition, user, category, createdAt } = data
+
+          return {
+            id,
+            term,
+            definition,
+            category: category.category,
+            username: user.username,
+            created_at: getUnixTimestamp(createdAt),
+          }
+        }),
+      })
+    } catch (error) {
+      return response.json({
+        code: 500,
+        status: 'Error',
+        message: 'Internal server error',
+      })
+    }
+  }
+
+  public async store({ request, response }: HttpContextContract) {
+    const DEFAULT_STATUS_DEFINITION_ID = 1
+
+    try {
+      const payload = await request.validate(CreateDefinitionValidator)
+      const validData = {
+        ...payload,
+        statusDefinitionId: DEFAULT_STATUS_DEFINITION_ID,
+      }
+      const newDefinition = await Definition.create(validData)
+
+      return response.status(201).json({
+        code: 200,
+        status: 'Success',
+        message: 'Definition created',
+        data: newDefinition,
```

Comment:
reformat return data, created_at and updated_at must be in unix_timestamp format